project/plugins.sbt

```diff
@@ -1,10 +1,13 @@
 
 resolvers += Resolver.sonatypeRepo("public")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.1")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15")
```

Comment:
I'll remove it for now from the PR, but I'm curious to understand the comment - I should try out mecha soon.